src/views/portals/outings-stats/outings-stats.js

```diff
@@ -126,35 +126,42 @@ export class Histogram {
 }
 
 export class StackedHistogram extends Histogram {
-  constructor(data, htmlElement, getX, getCategory) {
+  constructor(data, htmlElement, getX, getCategories) {
     super(data, htmlElement, getX);
-    this._getCategory = getCategory;
+    this._getCategories = getCategories;
+    this._getCategoryLabel = (category) => category;
     this._color = () => '#F93';
-    this._categoryComparator = (a, b) => (a < b ? -1 : a > b ? +1 : 0);
+    this._categoryComparator = (a, b) => (a < b ? -1 : a > b ? 1 : 0);
```

Comment:
Are you sorting strings? Why do you want to reinvent the wheel? https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
So, actually, you're right, we use only string categories for the moment. BUT I used this on purpose: it's more generic as it works with any type thats allow `<` and `>` operators.